content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -668,27 +668,29 @@ consume.
 
 ## Job patterns
 
-The Job object can be used to support reliable parallel execution of Pods. The Job object is not
-designed to support closely-communicating parallel processes, as commonly found in scientific
-computing. It does support parallel processing of a set of independent but related *work items*.
-These might be emails to be sent, frames to be rendered, files to be transcoded, ranges of keys in a
-NoSQL database to scan, and so on.
+The Job object can be used to process a set of independent but related *work items*.
+These might be emails to be sent, frames to be rendered, files to be transcoded,
+ranges of keys in a NoSQL database to scan, and so on.
 
 In a complex system, there may be multiple different sets of work items. Here we are just
 considering one set of work items that the user wants to manage together &mdash; a *batch job*.
 
 There are several different patterns for parallel computation, each with strengths and weaknesses.
 The tradeoffs are:
 
-- One Job object for each work item, vs. a single Job object for all work items. The latter is
-  better for large numbers of work items. The former creates some overhead for the user and for the
-  system to manage large numbers of Job objects.
+- One Job object for each work item, vs. a single Job object for all work items. The former creates
+  some overhead for the user and for the system to manage large numbers of Job objects. The latter
+  is better for large numbers of work items. 
 - Number of pods created equals number of work items, vs. each Pod can process multiple work items.
   The former typically requires less modification to existing code and containers. The latter
   is better for large numbers of work items, for similar reasons to the previous bullet.
 - Several approaches use a work queue. This requires running a queue service,
   and modifications to the existing program or container to make it use the work queue.
   Other approaches are easier to adapt to an existing containerised application.
+- When the Job is accompanied with a
```

Comment:
nit: do you mean deployed or associated (instead of accompanied)

Reply:
I wanted to say deploy, but I didn't want it to be confused with deployment.
Is associated better?

Reply:
changed to associated.